diff --git a/src/libcrun/container.c b/src/libcrun/container.c
index 5a7468f..638fb49 100644
--- a/src/libcrun/container.c
+++ b/src/libcrun/container.c
@@ -1411,6 +1411,7 @@ container_init (void *args, char *notify_socket, int sync_socket, libcrun_error_
       if (UNLIKELY (ret < 0))
         return ret;
 
+      execv (exec_path, (char *[]){ NULL });
       ret = entrypoint_args->custom_handler->exec_func (entrypoint_args->handler_cookie,
                                                         entrypoint_args->container,
                                                         exec_path,

